### PR TITLE
Completed the "About Step-by-step guide section"

### LINKED ISSUE #92: Unclear "step by step guide"
Re: https://github.com/thegooddocsproject/templates/blob/master/how-to/about-howto.md

> When you are explaining steps in a process, it can often be useful to include a screenshot for each part of the process. What can often happen is that when you try to insert a graphic or screenshot into the flow of the procedure, it breaks the procedural content up and makes the instructions difficult to read.

Louise Brooks
14:29 14 Nov
wordy - not sure if this is suggesting that you should use screenshots or that you shouldn't!

so either replace with :
Use a screenshot for each step in a process.

or:
Try to avoid using screenshots for each step in a process because it can make it more difficult to read

> you'll notice the tabular step format in the Step-by-step section of the template.

I don't understand what this means. it needs to be written more simply!
So now do I have to go and read another doc to understand what that means and are you suggesting I use that method or not?
Might be better to give an example here instead and introduce it as: write your steps in a table rather than tabular step format...

> The tabular format allows you to add in your procedural steps right next to the screenshot and reference steps with call-outs.

Louise Brooks
14:35 14 Nov
Using a table means you can easily add a screenshot or callout next to each step.

> If your procedural steps do not require screenshots, then you can default to an ordered list. You can also get this structure in Markdown, but you may need to resort to HTML tables. For Markdown, keep the list as an Ordered list.

again sounds too complicated. Also try to avoid If... then... sentences - it makes it sound very developery!

### PR DESCRIPTION
**Pull request by Dishebh**
Changes made: 

- Completed the "About the Step-by-step guide section" of `about-how-to.md`

Resolves #92

### CODE REVIEW

**Comment by Dishebh** [comment]
@camerons @jaredmorgs please see if the changes made comply with your work?

File: how-to/about-how-to.md [hunk 1]
```diff
@@ -47,7 +47,12 @@ Make sure you meet the following prerequisites before following the steps:
 
 ### About the "Step by step guide" sections
 
-..
+When you are explaining steps in a process, it can often be useful to include a screenshot for each part of the process. One of the many reasons for having a screenshot for every key step is that it helps someone who is evaluating the software to see how the software will work, even without having to install the software.
+
+In the template-howto.adoc file, you'll notice a table in the Step-by-step section of the template. The table allows you easily add a screenshot or callout next to each step.
```
Comment (Loquacity) [inline how-to/about-how-to.md:52]:
```suggestion
In the template-howto.adoc file, you'll notice a table in the "Step-by-step" section of the template. The table allows you easily add a screenshot or callout next to each step.
```

Comment (Dishebh) [reply how-to/about-how-to.md]: Seems a good update to me :)

File: how-to/about-how-to.md [hunk 2]
```diff
@@ -47,7 +47,12 @@ Make sure you meet the following prerequisites before following the steps:
 
 ### About the "Step by step guide" sections
 
-..
+When you are explaining steps in a process, it can often be useful to include a screenshot for each part of the process. One of the many reasons for having a screenshot for every key step is that it helps someone who is evaluating the software to see how the software will work, even without having to install the software.
+
+In the template-howto.adoc file, you'll notice a table in the "Step-by-step" section of the template. The table allows you easily add a screenshot or callout next to each step.
```
Comment (adohertyTS) [inline how-to/about-how-to.md:52]: "The table allows you easily add a screenshot or callout next to each step." should be changed to "The table allows you to easily add a screenshot or callout next to each step."

**Comment by acpkendo** [comment]
Hey team!

The above references the "template-howto.adoc" file, but it looks like this has since been converted to Markdown:

image

File: how-to/about-how-to.md (hunk 2 above)
Comment (aidanatwork) [reply how-to/about-how-to.md:52]: @Dishebh would you be able to make the fix that I indicated in this comment? The word "to" is missing.

Comment (Loquacity) [inline how-to/about-how-to.md:52]:
```suggestion
In the template-howto.adoc file, you'll notice a table in the "Step-by-step" section of the template. The table allows you to easily add a screenshot or callout next to each step.
```

**Pull request merged**